pino.js

```diff
@@ -170,7 +184,7 @@ function Pino (opts, stream) {
 }
 
 Pino.prototype = new EventEmitter()
-defineLevelsProperty(Pino.prototype)
+defineLevelsProperty(Pino.prototype, defaultLevelsValues)
```

Comment:
yeah child creation isn't a big deal but I'd rather not slow down the benchmarks either hmmmmm